presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -1883,6 +1883,10 @@ private static Domain buildColumnDomain(ColumnHandle column, List<HivePartition>
         Table table = metastore.getTable(tableName.getSchemaName(), tableName.getTableName())
                 .orElseThrow(() -> new TableNotFoundException(tableName));
 
+        if (table.getStorage().getBucketProperty().isPresent() && !isHiveBucketingV1(table)) {
```

Comment:
Nit: it would be reasonable to consult `table.getStorage().getBucketProperty().get()` on the RHS of `&&`

(just to consider)

Reply:
Done